components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/PlacesHistoryStorage.kt

```diff
@@ -47,6 +48,9 @@ open class PlacesHistoryStorage(
     override val logger = Logger("PlacesHistoryStorage")
 
     override suspend fun recordVisit(uri: String, visit: PageVisit) {
+        if (!canAddUri(uri)) {
```

Comment:
Should we log (debug) here that we're not recording? Right now we silently do nothing for those URIs, but that was already the case before so not sure if worth it?

Reply:
Oh, yes. We talked about it but I forgot to actually add the log statement(s). I think a debug log would be helpful in case someone's hits a case of some history missing and is able to provide us with logs.